demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1517,7 +1521,7 @@ def execute_test(self, proxy: Optional[MITMProxy] = None) -> bool:
                 test_executed = self._execute_unmockable_test()
             return test_executed
         except Exception:
-            self.build_context.logging_module.exception(f'Failed to execute {self.playbook}')
+            self.build_context.logging_module.exception(f'Test failed: {self}')
             self.build_context.tests_data_keeper.failed_playbooks.add(self.playbook.configuration.playbook_id)
```

Comment:
what's the difference between this and `self._add_to_failed_playbooks`?

Reply:
exception logging is the only difference.